views/AppView.js

```diff
@@ -0,0 +1,53 @@
+var AppView = Backbone.View.extend({
+  el: 'body',
+
+  events: {
+    'click #beer-submit': 'addBeer',
+    'click .sort': 'sort'
+  },
+
+  initialize: function(){
+    this.listenTo(this.model.get('beerCollection'), 'add', this.renderBeer);
+    this.listenTo(this.model.get('beerCollection'), 'sort', this.reRender);
+    this.$beerContainer = this.$('#beer-container');
+    this.model.get('beerCollection').comparator = "avgRate";
+  },
+
+  renderBeer: function(beer){
+    var view = new BeerView({model: beer});
+
+    this.$beerContainer.append(view.render().el);
+  },
+
+  addBeer: function(e){
+    var $form = $(e.target).parent();

```

Comment:
Why use `e.target` here? We know that our input lives as a child of this view's `$el`.